### DISCUSSION TITLE
Postgres array type is not supported

### DISCUSSION THREAD

**Discussion by demget** [directus/directus#6981] [Archive]
Why is there no Postgres (or even some other relational database) `array` type support? It recognizes by Directus as unknown and is not included in the data type superset as it was in `v8`. Is it a missed feature or one to be implemented later?

I can't even update the row because of an invalid representation of the array field which is set outside Directus, as well as hide it or made as read-only. It just says the type is not supported.

**Comment by rijkvanzanten** [comment]
Array types don't exist in every database vendor, and aren't supported by `knex` (the db abstraction layer we rely on) neither. V8s `array` was just a simple CSV stored as a string, which still exists under the name CSV.

**Reply by demget** [reply]
Well, then it's impossible to use Directus along with Postgres fully, sadly though... I have found some workarounds in `knex` to make arrays work, so gonna fork and try to implement their support by myself. Thank you anyway.
